# HG changeset patch
# User andrew
# Date 1250728138 -3600
# Node ID 8001ba2bf10d8f6f9f26bf3684e9fd06ddb145ba
# Parent  175bd68779546078dbdb6dacd7f0aced79ed22b1
6873059: Explicitly use -source 6 -target 6 when compiling with the boot jdk javac
Summary: The bootstrap javac currently uses the default source and targets of the boot javac
Reviewed-by: jjg, ohair

diff -r 175bd6877954 -r 8001ba2bf10d make/common/shared/Defs-java.gmk
--- openjdk.orig/corba/make/common/shared/Defs-java.gmk	Fri Aug 14 08:49:25 2009 -0700
+++ openjdk/corba/make/common/shared/Defs-java.gmk	Thu Aug 20 01:28:58 2009 +0100
@@ -179,7 +179,9 @@
 ifeq ($(COMPILER_WARNINGS_FATAL), true)
   BOOT_JAVACFLAGS  += -Werror
 endif
-BOOT_JAVACFLAGS  += -encoding ascii
+BOOT_SOURCE_LANGUAGE_VERSION = 6
+BOOT_TARGET_CLASS_VERSION = 6
+BOOT_JAVACFLAGS  += -encoding ascii -source $(BOOT_SOURCE_LANGUAGE_VERSION) -target $(BOOT_TARGET_CLASS_VERSION)
 
 BOOT_JAVA_CMD      = $(BOOTDIR)/bin/java $(JAVA_TOOLS_FLAGS)
 BOOT_JAVAC_CMD     = $(BOOTDIR)/bin/javac $(JAVAC_JVM_FLAGS) $(BOOT_JAVACFLAGS)
